Data/Sys/GameSettings/RODE01.ini

```diff
@@ -25,6 +25,5 @@ PH_ExtraParam = 0
 PH_ZNear =
 PH_ZFar =
 
-[Video_Hacks]
-EFBEmulateFormatChanges = True
-
+[Video_Settings]
+AspectRatio = 2
```

Comment:
Please remove the AspectRatio setting as that is a user preference and not something that the emulator should preset.